scripts/build_installer.py

```
@@ -172,63 +124,67 @@ def build_installer(
-        raise EvaluationBuildError("InstallBuilder was detected using an evaluation version.")
-    elif local_dev_build and EVALUATION_VERSION_STRING not in output:
+    if EVALUATION_VERSION_STRING in output.stdout and not local_dev_build:
         raise EvaluationBuildError(
-            "InstallBuilder was not detected using an evaluation version when running a dev build. "
+            "InstallBuilder was detected using an evaluation version, which is only permitted in local dev builds."
+        )
+    elif local_dev_build and EVALUATION_VERSION_STRING not in output.stdout:
+        print(
+            "WARNING: InstallBuilder was not detected using an evaluation version when running a dev build. "
             "This could indicate that the error messaging when using an evaluation version has changed.\n"
             "Please check the InstallBuilder logs to confirm if the error messaging has changed from "
             f"'{EVALUATION_VERSION_STRING}' and update the install_builder.py script accordingly."
         )
     return out_dir
 
 
-def dev_create_dcc_component(workdir: tempfile.TemporaryDirectory, dcc_component: DccSubmitter):
+def dev_create_dcc_component(
+    workdir: tempfile.TemporaryDirectory, dcc_component: DccSubmitter
+) -> None:
     """
     Creates artifacts locally
     """
     # Clone dcc component by copying it into the working directory & allowing the dependency bundle script to be executed
     repo_dir = f"{workdir}/{dcc_component.componentName}"
-    source_folder = Path(os.path.abspath(__file__)).parent.parent
+    source_folder = Path(__file__).absolute().parent.parent
     shutil.copytree(source_folder, repo_dir, dirs_exist_ok=True)
-    bundle_file = Path(os.path.join(repo_dir, "depsBundle.sh"))
+    bundle_file = Path(repo_dir) / "depsBundle.sh"
     bundle_file.chmod(bundle_file.stat().st_mode | stat.S_IEXEC)
-    run(str(bundle_file))
+    subprocess.run(str(bundle_file))
```

Should we do `check=True` here? Otherwise this can fail silently.